e2e/src/account/app.page.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright (c) 2018 THREEANGLE SOFTWARE SOLUTIONS SRL
+ * Available under MIT license webFrame/LICENSE
+ */
+
+import { IAccountCredentials, PAGE_URL } from 'app-shared/core';
+import { browser, by, Key, WebElement } from 'protractor';
+
+export class AppPage {
+  public async openHomePage(): Promise<void> {
+    return browser.get(PAGE_URL.HOME_PAGE);
+  }
+
+  public async clickLogoutButton(): Promise<void> {
+    await browser.findElement(by.name('logout')).click();
+  }
+
+  public async getPageTitle(): Promise<string> {
+    return browser.findElement(by.css('app-root h1.app-page__title')).getText();
+  }
+
+  public async openLoginPage(): Promise<void> {
+    return browser.get(PAGE_URL.LOGIN_PAGE);
+  }
+
+  public async openSignupPage(): Promise<void> {
+    return browser.get(PAGE_URL.SIGNUP_PAGE);
+  }
+
+  public async openForgotPasswordPage(): Promise<void> {
+    return browser.get(PAGE_URL.FORGOT_PASSWORD_PAGE);
+  }
+
+  public async setFormData(
+    formEl: WebElement,
+    data: Record<string, string>,
+  ): Promise<void> {
+    const inputNames: string[] = Object.keys(data);
+    // tslint:disable-next-line:prefer-for-of
+    for (let index = 0; index < inputNames.length; index++) {
+      const inputName: string = inputNames[index];
+      await formEl.findElement(by.name(inputName)).sendKeys(data[inputName]);
+    }
+  }
+
+  public async submitLoginCredentials(credentials: IAccountCredentials): Promise<void> {
```

Comment:
These should be in LoginPageTestability

Reply:
I've split the current test suite into 2 different test suites.